Realm/RLMObjectBase.h

```diff
@@ -33,6 +33,9 @@ RLM_ASSUME_NONNULL_BEGIN
 
 + (NSString *)className;
 
+/// Returns whether the class is included in the default set of classes persisted in a Realm.
```

Comment:
Either this needs to be made not a doc-comment, moved elsewhere, or RLMObjectBase.h needs to be addded to appledoc's ignore list, because RLMObjectBase should not appear in our docs.